coops-ui/src/components/dialogs/DashboardUnavailableDialog.vue

```diff
@@ -43,6 +42,9 @@ export default class DashboardUnavailableDialog extends Vue {
   // Prop to display the dialog.
   @Prop() private dialog: boolean
 
+  // Prop to provide attachment selector.
+  @Prop() private attach: string
```

Comment:
I don't fully understand the usage of this prop here and in the other dialogs. This is purely a question for my own knowledge, could i get the jist of why we pass this prop down? Is this to give the parent a reference to the dialogs real content, opposed to a just a reference to the existence of the child on the parent?

Reply:
Good question!

Although dialogs seem to live within a component, in fact the only thing there in the DOM is a stub. The actual dialog code/content is normally rendered dynamically right under `#app` (outside of the component). This is why it's hard to test dialogs, tooltips and similar.

But there exists the "attach" attribute, which causes the dialog content to now be dynamically rendered under its parent component. Now it can be selected easily! I did this in case QA wants to test any dialog.

Reply:
Great explanation. I'm just as happy as QA with this discovery as well. This will only improve and make our unit testing easier. well done sir!